frontend/docusaurus.config.cjs

```diff
@@ -0,0 +1,124 @@
+const lightCodeTheme = require("prism-react-renderer/themes/github");
+const darkCodeTheme = require("prism-react-renderer/themes/dracula");
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: "Shields.io",
+  tagline: "Concise, consistent, and legible badges",
```

Comment:
I don't have a strong position on this but will spend some time contemplating. My initial inclination is something along the lines of...

> yeah, we probably don't need to go out of our way to push them, but, we do still provide them because they are helpful in a few contexts and it's helpful for new users to know that we _do_ offer them

though i'm trying to challenge myself to gauge whether i'm just being biased given my prior work on squint

Reply:
The reason I said that specifically here is we currently [sometimes](https://github.com/badges/shields/issues/3654) use the tag line "Concise, consistent, and legible badges in SVG and raster format". I've dropped "in SVG and raster format" from that. I don't think it needs to be in the first line someone reads about the project. We don't necessarily need to completely stop documenting that png exists as an option though.

I can't remember if raster made its ways into this anywhere, but let me take an action to make sure it is in there somewhere.